src/pages/game/[id].tsx

```diff
@@ -53,24 +73,37 @@ const GameSetup: NextPage<Props> = ({ game }) => {
           tabIndex={-1}
         >
           {addons.data?.pages.map(page =>
-            page.items.map(x =>
-              <Addon key={x.id} addon={x} />
+            page.items.map(addon =>
+              <Addon
+                key={addon.id}
+                addon={addon}
+                active={activeAddons.has(addon.id)}
+                onClick={() => setActiveAddons(old => {
+                  const test = new Map(old);
+                  test.has(addon.id) ? test.delete(addon.id) : test.set(addon.id, addon);
+                  return test;
+                })
+                }
+              />
             )
           )}
         </section>
+        <div className={styles.fog}></div>
 
         {/* Card count and settings */}
         <section className={styles.horizontalList}>
-          <Tag><BsWifi />125</Tag>
-          <Tag><BsWifiOff />125</Tag>
-          <Button variant="secondary">Settings</Button>
+          <Tag><BsWifi />{cardSize.online}</Tag>
+          <Tag><BsWifiOff />{cardSize.offline}</Tag>
+          <Button variant="secondary" size="small">Settings</Button>
         </section>
 
         {/* Start */}
-        <section className={styles.horizontalList}>
-          <Button>Start Online</Button>
-          <Button>Start Offline</Button>
-        </section>
+        {activeAddons.size > 0 &&
```

Comment:
Is at least one addon required to start playing?

Reply:
atleast x amount of cards yes